udata/core/metrics/utils.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is it the better place ?
Maybe be we should introduce a tracking module in order to group tracking related helpers ?